submariner/IPV6-OVN.md

```diff
@@ -0,0 +1,105 @@
+# **Submariner OVN CNI Enhancement for IPv6 Support**
+
+## **Summary**
+
+This proposal outlines the changes required in Submariner for  OVN Kubernetes
+CNI to enable IPv6 support, ensuring seamless connectivity between clusters
+using Submariner. The main proposal has the full design
+[IPv6 Datapath Enhancements](IPV6-datapath.md)
+This covers only the OVN CNI part of it.
+
+---
+
+## Design Details
+
+The OVNKubernetes handler programs network policies and routes to direct traffic from
+the gateway and non-gateway nodes to direct the traffic to the remote cluster.
```

Comment:
Maybe we need to rephrase this sentence?

Reply:
Done , changed to driver instead of handler.

Reply:
```suggestion
the gateway and non-gateway nodes to the remote cluster.
```